docs/pages.jl

```diff
@@ -1,7 +1,7 @@
 # Put in a separate page so it can be used by SciMLDocs.jl
 
 pages = [
-    "Home" => "index.md",
+    "Overview" => "index.md",
```

Comment:
We do need some kind of tag line.

<img width="431" alt="Screenshot 2022-08-02 094516" src="https://user-images.githubusercontent.com/1814174/182389995-f0652ccc-013a-4f3d-b0e5-7087f873ac7f.png">
 
Otherwise it won't do well in search. This doesn't necessarily need to be the label in the SciMLDocs.

Reply:
I don't immediately see a way for the two to be separated?
The <title> element of https://nonlinearsolve.sciml.ai/ seems to be "first item on dropdown menu · NonLinearSolve.jl"
where NonLinearSolve.jl is what is used as the sitename keyword argument in the makedocs function.

Reply:
We can make a bool that is set before `pages.jl` is loaded that then allowed for `is_scimldocs` and to switch whether to place the title on or not.

But I would greatly prefer just always having the title so it shows up on search engines, and making an issue to Documenter.jl to decouple the sidebar name from the title.

Reply:
You can give a different sidebar name using that technique, but you still have to fill in something for the sidebar for `is_scimldocs=false`. 

Currently for DifferentialEquations.jl it is 

<title>DifferentialEquations.jl: Scientific Machine Learning (SciML) Enabled Simulation and Estimation · DifferentialEquations.jl</title>

Which has DifferentialEquations in there twice, once from the a name in the dropdown menu and once from the sitename keyword argument. 

So you could make the sitename argument an empty string. But then if you go to down the dropdown menu e.g

<title>Solving Large Stiff Equations · DifferentialEquations.jl</title>

It would also lose the DifferentialEquations, which is not great if somebody is searching for: "stiff equations DifferentialEquations.jl". 

I'll make an issue with Documenter.jl .